modules/enabling-baseline-capability-set.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_baseline_capability_set_{context}"]
+= Enabling the cluster capabilities by setting baseline capability set
+
+As a cluster administrator, you can enable the capabilities by setting `baselineCapabilitySet`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* To set the `baselineCapabilitySet`, run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type merge -p '{"spec":{"capabilities":{"baselineCapabilitySet":"vCurrent"}}}' <1>
+----
++
+<1> For `baselineCapabilitySet` you can specify `vCurrent`, `v4.11`, or `none`.
```

Comment:
nit: [should be `None`][1].  I've filed #48207 with the fix, but haven't checked about backports.

[1]: https://github.com/openshift/api/blob/d2198688dc209e0ed3a6ab1c74ec02cf36b64cf3/config/v1/types_cluster_version.go#L258-L264